### PR TITLE
SLING-8419 JSON serializer for configurations

### PR DESCRIPTION
**Pull request by kwin**
Config serializers are now part of installer core so that all providers
can just use them.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,84 +38,89 @@
...
                         </Bundle-Activator>
                         <Conditional-Package>
                             org.apache.felix.configurator.impl.json,
-                            org.apache.felix.configurator.impl.model                            
+                            org.apache.felix.configurator.impl.model,
+                            org.apache.sling.feature,
+                            org.apache.sling.feature.*
                         </Conditional-Package>
                         <Embed-Dependency>
-                            geronimo-json_1.0_spec,johnzon-core,org.apache.felix.converter,
```
Comment (kwin) [inline pom.xml:63]: @cziegeler Was there a reason JSON-P was embedded? I ran into the issue mentioned at https://issues.apache.org/jira/browse/GERONIMO-6560 with these dependencies embedded! Was it only there to keep the dependencies to a minimum? Just relying on the bundle https://github.com/apache/sling-org-apache-sling-commons-johnzon should be fine IMHO.

Comment (cziegeler) [reply pom.xml:63]: yes, the reason was to keep the dependencies to a minimum.

Comment (kwin) [reply pom.xml:63]: Either we need to rely on Aries SPI Fly or embed Sling Commons JSON, otherwise we run into classloader issues.

Comment (cziegeler) [reply pom.xml:63]: and if we just make it a regular dependency?

Comment (kwin) [reply pom.xml:63]: I think embedding is fine (https://github.com/apache/sling-org-apache-sling-installer-core/pull/5/commits/9cf63dfb448729d91272b14c5979c632ce1f046b). The total size of the bundle is now 400kB. Currently in Sling Johnzon is loaded with StartLevel 5 (https://github.com/apache/sling-org-apache-sling-starter/blob/9898f313147e97e15721b71198c5196d4b4a1262/src/main/provisioning/sling.txt#L42)

**Pull request merged**